docs/src/main/mdoc/datatypes/freemonad.md

```diff
@@ -525,31 +525,31 @@ type TeletypeState[A] = State[List[String], A]
 /** Teletype smart constructors */
 object TeletypeOps {
   def writeLine(line : String) : TeletypeT[TeletypeState, Unit] =
-	FreeT.liftF[Teletype, TeletypeState, Unit](WriteLine(line))
+  FreeT.liftF[Teletype, TeletypeState, Unit](WriteLine(line))
   def readLine(prompt : String) : TeletypeT[TeletypeState, String] =
-	FreeT.liftF[Teletype, TeletypeState, String](ReadLine(prompt))
+  FreeT.liftF[Teletype, TeletypeState, String](ReadLine(prompt))
   def log(s : String) : TeletypeT[TeletypeState, Unit] =
-	FreeT.liftT[Teletype, TeletypeState, Unit](State.modify(s :: _))
+  FreeT.liftT[Teletype, TeletypeState, Unit](State.modify(s :: _))
 }
 
 def program : TeletypeT[TeletypeState, Unit] = {
   for {
-	userSaid <- TeletypeOps.readLine("what's up?!")
-	_ <- TeletypeOps.log(s"user said : $userSaid")
-	_ <- TeletypeOps.writeLine("thanks, see you soon!")
+  userSaid <- TeletypeOps.readLine("what's up?!")
+  _ <- TeletypeOps.log(s"user said : $userSaid")
+  _ <- TeletypeOps.writeLine("thanks, see you soon!")
   } yield ()
 }
 
 def interpreter = new (Teletype ~> TeletypeState) {
   def apply[A](fa : Teletype[A]) : TeletypeState[A] = {
-	fa match {
-	  case ReadLine(prompt) =>
-		println(prompt)
-		val userInput = "hanging in here" //scala.io.StdIn.readLine()
-		StateT.pure[Eval, List[String], A](userInput)
-	  case WriteLine(line) =>
-		StateT.pure[Eval, List[String], A](println(line))
-	}
+  fa match {
```

Comment:
this looks weird. It looks like the indentation is constant. Is it due to mixed tabs/spaces? Can we have a way that this compiles but also doesn't look weird in text?